Metalama.Framework.DesignTime/Pipeline/DesignTimeAspectPipeline.cs

```diff
@@ -886,7 +887,10 @@ public bool IsCompileTimeSyntaxTreeOutdated( string name )
             return null;
         }
 
-        var symbolId = symbol.GetSerializableId();
+        if ( symbol.TryGetSerializableId( out var symbolId ) )
```

Comment:
```suggestion
        if ( !symbol.TryGetSerializableId( out var symbolId ) )
```

Reply:
Quite strange that no tests broke because of that.

Reply:
Yeah, I've messed up splitting the tests from the fix.